src/Model/Constants.cs

```diff
@@ -59,7 +59,7 @@ internal static class Constants
         internal const string FillInNotes= "{{ Fill in the Notes }}";
 
         internal const string FillInExampleTitle = "Example 1";
-        internal const string FillInExampleCode = @"PS C:\> {{ Add example code here }}";
+        internal const string FillInExampleCode = @"{{ Add example code here }}";
         internal const string FillInExampleDescription = "{{ Add example description here }}";
         internal const string FillInParameterDescriptionTemplate = "{{{{ Fill {0} Description }}}}";
         internal const string FillInReleatedLinks = "{{ Fill in the related links here }}";
```

Comment:
Is it an intension to use "Releated" in the name and "related" in the text?

Reply:
Good catch. Fixing it.